.github/workflows/ci.yaml

```diff
@@ -0,0 +1,44 @@
+name: Continuous Integration
+
+on:
+    push:
+        branches: [master]
+    pull_request:
+        branches: [master]
+
+jobs:
+    test:
+        strategy:
+            matrix:
+                os: [ubuntu-latest, macos-latest, windows-latest]
+
+        runs-on: ${{ matrix.os }}
+
+        steps:
+            - uses: actions/checkout@v2
+            - name: Use Node.js 12
+              uses: actions/setup-node@v1
+              with:
+                  node-version: 12
+            - run: yarn install
+            - name: Test
+              uses: GabrielBB/xvfb-action@v1.0
+              with:
+                  run: yarn test
```

Comment:
I think there is a missing `download-binaries` action.
Or maybe we should add the following scripts in package.json:

```json
{
  "ci-test": "yarn run download-binaries && yarn test"
}
```